app/assets/javascripts/graphs.coffee

```diff
@@ -0,0 +1,155 @@
+$ ->
+  if gon.raw_run isnt `undefined`
+    pb_graph_data = ["PB - Gold"]
+    pb_graph_ticks = []
+    $.each gon.run.splits, (i, split) ->
+      time = split.duration - split.best
+      if time > 5
+        pb_graph_data.push d3.round(time, 2)
+        pb_graph_ticks.push split.name
+    pb_graph = c3.generate({
+      bindto: "#pb-graph",
+      title: { text: "PB vs Golds" },
+      data: {
+        columns: [pb_graph_data],
+        type: "bar"
+      },
+      axis: {
+        x: {
+          type: "category",
+          categories: pb_graph_ticks,
+          tick: {
+            rotate: -45,
+            multiline: false
+          },
+          height: 80
+        },
+        y: {
+          label: {
+            text: "Possible Time Save",
+            position: "outer-middle"
+          },
+          tick: {
+            format: (d) -> return d + "s"
+          }
+        }
+      },
+      legend: { hide: true }
+    })
+
+    if gon.raw_run.program isnt "livesplit"
+      return
+
+    split_history_data = ["Mean - Gold"]
+    split_history_ticks = []
+    $.each gon.raw_run.splits, (i, split) ->
+      if split.history is [] or split.history is null
+        return
+      mean = d3.round(d3.mean(split.history) - split.best, 2)
```

Comment:
The second graph is meant to show consistency.  By averaging the split history you can see how far off from you golds to give you splits that are wildly inconsistent that you might want to work on.